pixeltable/env.py

```diff
@@ -1047,7 +1048,10 @@ class PackageInfo:
     version: list[int] | None = None  # installed version, as a list of components (such as [3,0,2] for "3.0.2")
 
 
-TIME_FORMAT = '%H:%M.%S %f'
+TIME_FORMAT = '%m/%d/%Y, %H:%M:%S.%f %Z'
```

Comment:
In that case, do we really need some bespoke format here (versus just the usual `str()` output of a `datetime`?)